pcre_test.go

```diff
@@ -290,3 +290,34 @@ func TestFreeRegexp(t *testing.T) {
 	// Check that double calls don't fail.
 	re.FreeRegexp()
 }
+
+func TestFindAll(t *testing.T) {
+	re := MustCompile("\\d{2}x", 0)
+	data := "12x 12332xf 43bx62x"
+	expected := []Match{
+		Match{"12x", []int{0, 3}},
+		Match{"32x", []int{7, 10}},
+		Match{"62x", []int{16, 19}},
+	}
+	matches := re.FindAll(data, 0)
+	if len(matches) != 3 {
+		t.Error("Expected exactly 3 matches")
+	}
+
+	for i := 0; i < len(expected); i++ {
+		if matches[i].Finding != expected[i].Finding {
```

Comment:
Could use https://golang.org/pkg/reflect/#DeepEqual instead of checking each field.

Reply:
Fixed